### PR TITLE
Added homework_16_240123

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework16?

Thanks in advance!

### CODE REVIEW

File: homework_16_240123/src/MyArrayList.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+     *
+     * @return {@code true} if list is empty, {@code false} otherwise
+     */
+    public boolean isEmpty() {
+        if (size == 0) {
+            return true;
+        } else {
+            return false;
+        }
+//        return size == 0; // or so it is possible in one line
```
Comment (YeaaaaH) [inline homework_16_240123/src/MyArrayList.java:155]: size == 0 is also correct and more elegant

File: homework_16_240123/src/MyArrayList.java [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    public int size() {
+        return size;
+    }
+
+    /**
+     * Removes all list elements
+     */
+    public void clear() {
+        size = 0;
+        arrayElements = new Object[DEFAULT_CAPACITY];
```
Comment (YeaaaaH) [inline homework_16_240123/src/MyArrayList.java:170]: default capacity == 10; you need to make it empty

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need to fix

File: homework_16_240123/src/MyArrayList.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_16_240123/src/MyArrayList.java:155]: Fixed.

File: homework_16_240123/src/MyArrayList.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_16_240123/src/MyArrayList.java:170]: Fixed.
Changed: private static final int DEFAULT CAPACITY = 10;
Field DEFAULT CAPACITY is final and is not changeable.
If changed like this: arrayElements = new Object[0];
then we won't be able to add any element to this ArrayList

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**